### ISSUE TITLE
Who should I e-mail if I find a problem weaknesses/bugs?

### ISSUE THREAD

**Issue by shenlanting** [google/protobuf#4171]
We found some weaknesses/bugs. Please give an e-mail address, so I can send you the details.

tks.

**Comment by xfxyjwf** [comment]
We don't use private/restricted mailing lists for bug reports. If you think the bug is severe enough to not disclose to the public, can you summarize what the bug is about so we can assess whether it makes sense to establish a private communication channel?

**Comment by shenlanting** [comment]
No need to establish a private communication channel.
The thing is we need to check whether our tool for static analysis is good enough to find bugs besides coding weakness. I have sent a email, but seems not useful for you project. sorry for the interruption. We will continue improving our static analysis tool, hoping that it will be helpful for finding bugs one day.
